template/YYYY-MM-design-template.md

```diff
@@ -65,9 +65,38 @@ link to that issue instead.
 
 #### User story 2
 
+## User Experience (if applicable)
```

Comment:
What is this difference between this and the user scenarios section?

Reply:
Good question!

The user scenario section is expected to provide background on *when and why* a user would need this functionality. For example: https://github.com/radius-project/design-notes/blob/d6b3ea8c44905b5e6eac7b328ab33869e488b41e/recipe/2023-08-garbage-collection.md?plain=1#L37.

The user experience section aims to provide insights into *how* a user would interact with this functionality, outlining the proposed end to end user flow.

Please feel free to suggest any updates if this can be made clearer in the template to avoid any confusion. :)

I expect both of these sections to be covered as part of the product specification. The design doc should primarily copy them over from the product spec document going forward. cc @willtsai